service/src/main/java/com/ge/predix/acs/attribute/readers/ExternalAttributeReader.java

```diff
@@ -29,20 +55,78 @@ public ExternalAttributeReader(final AttributeCache attributeCache) {
                 attributes = this.getAttributesFromAdapters(identifier);
                 this.attributeCache.setAttributes(identifier, attributes);
 
-                // TODO: Reset the TTL for the policy evaluation cache key concerning this resource to:
-                // total customer TTL - this.attributeCache.getConfiguredTtlInSeconds()
+                //long policyEvalDecisionCacheTtlInSeconds =
```

Comment:
Implemented this as discussed. A related test hasn't been added yet because it would be time-based and thus error prone.